tox.ini

```diff
@@ -886,7 +886,8 @@ commands =
     bash -c 'k3d --version'
 
     ; create registry
-    bash -c 'k3d registry create registry.localhost --port 5800 -v $HOME/.k3d-registry:/var/lib/registry || true'
```

Comment:
local path `$HOME/.k3d-registry` gets mounted on the docker macos VM through virtiofs. Might be a reason why docker push gets stuck at some point. For now just use docker-native volume to make sure there's no virtiofs involved happening during a push.